README.md

```diff
@@ -1,6 +1,6 @@
-<h1 align="center">Live Share Chat</h1>
+<h1 align="center">Chat</h1>
 
-<h3 align="center">Chat with your team while you collaborate over code using VS Live Share</h3>
```

Comment:
@lostintangent, @fubaduba: Can you fix up the readme?

Reply:
Yup.